src/Std/Time/Duration.lean

```diff
@@ -0,0 +1,79 @@
+/-
+Copyright (c) 2024 Lean FRO, LLC. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Sofia Rodrigues
+-/
+prelude
+import Std.Time.Internal
+import Std.Time.Time
+
+namespace Std
+namespace Time
+open Internal
+
+/--
+`Instant` represents a place in time with second and nanoseconds precision.
+-/
+structure Instant where
```

Comment:
I'm afraid I still don't understand what this structure represents. Which point on the time line does the instant `0` represent. How do I move from `Timestamp` or `ZonedDateTime` to `Instant`?

Reply:
This type was created to represent an instant that was not bounded to an epoch, to give it meaning, it had to be used with another instant and create a duration, similar to how it's done in Rust. However, it I think that we can shift the objective and change it to be like a Timestamp that provides nanosecond precision.